crates/sui-indexer-alt/src/models/watermarks.rs

```diff
@@ -42,6 +42,24 @@ pub struct ReaderWatermark<'p> {
     pub reader_lo: i64,
 }
 
+#[derive(Queryable, Debug, Clone, FieldCount)]
+#[diesel(table_name = watermarks)]
+pub struct PrunerWatermark<'p> {
+    /// The pipeline in question
+    pub pipeline: Cow<'p, str>,
+
+    /// How long to wait from when this query ran on the database until this information can be
+    /// used to prune the database. This number could be negative, meaning no waiting is necessary.
+    pub wait_for: i64,
+
+    /// The pruner can delete up to this checkpoint, (exclusive).
+    pub reader_lo: i64,
+
+    /// The pruner has already deleted up to this checkpoint, so can continue from this point
+    /// (inclusive).
+    pub pruner_hi: i64,
```

Comment:
```suggestion
    pub pruner_hi_inclusive: i64,
```

Reply:
It's actually an exclusive upperbound (and an inclusive lowerbound). I'll make the doc comment clearer.